libfwupdplugin/fu-device.c

```diff
@@ -456,6 +456,31 @@ fu_device_get_parent (FuDevice *self)
 	return priv->parent;
 }
 
+/**
+ * fu_device_get_root:
```

Comment:
get root makes me think this is returning the parent of a parent (like the very top chain), but it's just returning the parent of the device.

Reply:
It really is getting the parent of the parent.